samples/quarkus-with-restassured/src/main/java/org/rodnalsol/sample/quarkuswithrestassured/rest/ExampleRest.java

```diff
@@ -0,0 +1,61 @@
+package org.rodnalsol.sample.quarkuswithrestassured.rest;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.rodnalsol.sample.quarkuswithrestassured.dto.ErrorResponse;
+import org.rodnalsol.sample.quarkuswithrestassured.dto.UserRequest;
+import org.rodnalsol.sample.quarkuswithrestassured.dto.UserResponse;
+
+/**
+ * Example REST for openapi-extender-restassured module.
+ *
+ * @author csaba.balogh
+ * @since 0.3.0
+ */
+@Path("/example")
+public class ExampleRest {
```

Comment:
Let's have a chat about the demo classes and their internals. Probably not looking good to put silly things into them, but I know I started it :D